mcs/resource_mcs_db_cluster.go

```diff
@@ -140,7 +140,7 @@ func resourceDatabaseCluster() *schema.Resource {
 							ForceNew: true,
 							ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
 								v := val.(string)
-								if v != Galera && v != Postgres {
+								if v != Galera && v != Postgres && v != Tarantool {
```

Comment:
i guess construction like
```
switch v {
    case Galera, Postgres, Tarantool: // do nothing
    default: errs = .... // catch error 
}
```
would be more pleasant for an eye

Reply:
I think we can refactor this whole func to use terraform provided validation.StringInSlice: https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/validation#StringInSlice

`
ValidateFunc: validation.StringInSlice(getClusterDatastores(), true)
`

Reply:
an option, agree